docs/docs/installation.md

```diff
@@ -20,7 +20,9 @@ If you're just interested in playing around with React, you can use CodePen. Try
 
 If you prefer to use your own text editor, you can also <a href="/react/downloads/single-file-example.html" download="hello.html">download this HTML file</a>, edit it, and open it from the local filesystem in your browser. It does a slow runtime code transformation, so don't use it in production.
 
-## Creating a Single Page Application
+Currently there are two popular ways to get started with React:
+
+## 1. Using Create React App
```

Comment:
You are right, that's why I removed the line that says `Creating a Single Page Application` and instead added the line `Currently there are two popular ways to get started with React:`. Of course numbering isn't always the best solution, it was just my suggestion for a quick 'fix' ;)